etc/ctrl.with.edge.yml

```diff
@@ -75,10 +75,10 @@ events:
     subscriptions:
 #      - type: fabric.routers
 #      - type: fabric.terminators
-      - type: metrics
-        sourceFilter: .*
-        metricFilter: .*pool.*
-#      - type: fabric.circuits
+#      - type: metrics
+#        sourceFilter: .*
+#        metricFilter: .*pool.*
+      - type: fabric.circuits
```

Comment:
Was this intentional?

Reply:
Now that we have the config generation, I haven't been keeping these files as tidy. Not sure if it matters. I'll try and tidy it back up on the next update